Sources/SpotHeroAPI/Search/Models/MonthlyRate.swift

```diff
@@ -0,0 +1,45 @@
+// Copyright © 2020 SpotHero, Inc. All rights reserved.
+
+import Foundation
+
+/// Monthly-specific metadata pertaining to a rate for the rental of a parking spot.
+public struct MonthlyRate: Codable {
+    private enum CodingKeys: String, CodingKey {
+        case amenities
+        case contract
+        case inOutPrivileges = "in_out_privileges"
+        case postPurchaseInstructions = "post_purchase_instructions"
+        case redemption
+        case reservationType = "reservation_type"
+        case reservationDates = "reservation_dates"
+        case startDateRestrictions = "start_date_restrictions"
+        case title
+    }
+    
+    /// Monthly parking amenities offered for this rate at the facility.
+    public let amenities: [Amenity]
+    
+    // WIP: Missing Docs
+    public let redemption: Redemption
+    
+    /// Human-readable description of the rate.
+    public let title: String
+    
+    /// Additional information to follow up with before parking at the facility.
+    public let postPurchaseInstructions: String
+    
+    /// This shows what day of the month that billing will start.
```

Comment:
Could this remove the "this shows" part?
```suggestion
    /// What day of the month that billing will start.
```

Just like for other properties, we don't have:
```
/// This shows additional information to follow up with before parking at the facility.
public let postPurchaseInstructions: String
```

Reply:
Yep easy enough.